core/includes/theme.inc

```diff
@@ -1948,18 +1971,14 @@ function theme_dropbutton_wrapper($variables) {
  *   - title: The title text is displayed when the image is hovered in some
  *     popular browsers.
  *   - attributes: Associative array of attributes to be placed in the img tag.
+ *     All attributes should be populated with the correct values that match
+ *     the other input variables.
+ *
+ * @since 1.8.0 Added support for $variables['path']. All variables are now
+ *   populated into $variables['attributes'] in template_preprocess_image().
  */
 function theme_image($variables) {
-  $attributes = $variables['attributes'];
-  $attributes['src'] = file_create_url($variables['uri']);
-
-  foreach (array('width', 'height', 'alt', 'title') as $key) {
-    if (isset($variables[$key])) {
-      $attributes[$key] = $variables[$key];
-    }
-  }
-
-  return '<img' . backdrop_attributes($attributes) . ' />';
+  return '<img' . backdrop_attributes($variables['attributes']) . ' />';
```

Comment:
backdrop_attributes() intentionally includes a prefix space (that way if no attributes were added you don't have an empty space in the tag).

https://api.backdropcms.org/api/backdrop/core%21includes%21common.inc/function/backdrop_attributes/1